remote/Dockerfile

```diff
@@ -34,8 +34,8 @@ RUN pip install --break-system-packages supervisor
 COPY supervisord.conf /etc/supervisord.conf
 
 # Allow any user to start the RDP Server
-RUN sed -i 's/allowed_users=console/allowed_users=anybody/g' /etc/X11/Xwrapper.config
-
+RUN sed -i 's/allowed_users=console/allowed_users=anybody/g' /etc/X11/Xwrapper.config \
+    || echo "allowed_users=anybody" > /etc/X11/Xwrapper.config
```

Comment:
Why is there an or statement here? I would expect one way of configuration.

Reply:
For some reason, the `/etc/X11/Xwrapper.config` doesn't exist in our internal Base image at this point. 
The statement should do: Replace `allowed_users=console` with `allowed_users=anybody` if the `/etc/X11/Xwrapper.config` file exists, otherwise create the file and add the `allowed_users=anybody` line. 

I can also do it explicitly via an if statement.

Reply:
I assume a comment above the command is sufficient. Something like: Depending on the base image used, Xwrapper.config may (not) be available.

Reply:
Done.